src/components/userProfile/components/styles.css

```diff
@@ -7,26 +7,26 @@
 .data-title {
     margin-top: 22px;
     margin-bottom: 19.2px;
+    font-weight: 600;
 }
 .data-title div {
      float: right;
-     font-size: 12.5px;
+     font-size: 13.5px;
      text-align: right;
+     font-weight: 300;
 }
-.data-title b {
-    font-size: 13.5px;
-}
+
 .basic-info {
     margin-top: 15px;
     padding-bottom: 17.4px;
+    font-weight: 600;
 }
-.basic-info b {
-    font-size: 13.5px;
-}
+
 .basic-info div {
     float: right;
-    font-size: 13px;
+    font-size: 13.5px;
```

Comment:
dont use decimal pixels. either leave it at `131 or move it up to another whole number

Reply:
Ok